CMakeLists.txt

```diff
@@ -387,11 +388,7 @@ if(SENTRY_LINK_PTHREAD)
 endif()
 
 # apply platform libraries to sentry library
-if(SENTRY_LIBRARY_TYPE STREQUAL "STATIC")
-	target_link_libraries(sentry PUBLIC ${_SENTRY_PLATFORM_LIBS})
-else()
-	target_link_libraries(sentry PRIVATE ${_SENTRY_PLATFORM_LIBS})
-endif()
+target_link_libraries(sentry PRIVATE ${_SENTRY_PLATFORM_LIBS})
```

Comment:
You want to export link libs for static linkage. `PRIVATE` does that.
`PUBLIC` is needed if include dirs etc. need to be used, too. This normally doesn't depend on the type of linkage but on the design of the headers.

Reply:
Honestly, that use of `PUBLIC` predates my involvement, and I don't know why it was added in detail. I guess the difference resolves to a `LINK_ONLY` generator expression around the interface link libraries in the exported sentry target, so switching to PRIVATE in both cases shouldn't break anything.